docs/pages/base/api/use-input.json

```
@@ -33,7 +33,59 @@
     },
     "value": { "type": { "name": "unknown", "description": "" } }
   },
-  "returnValue": {},
+  "returnValue": {
+    "disabled": {
+      "type": { "name": "boolean", "description": "If `true`, the component will be disabled." },
+      "default": "false",
+      "required": true
+    },
+    "error": {
+      "type": {
+        "name": "boolean",
+        "description": "If `true`, the `input` will indicate an error by setting the `aria-invalid` attribute."
+      },
+      "default": "false",
+      "required": true
+    },
+    "focused": {
+      "type": { "name": "boolean", "description": "If `true`, the `input` will be focused." },
+      "default": "false",
+      "required": true
+    },
+    "formControlContext": {
+      "type": {
+        "name": "FormControlUnstyledState | undefined",
+        "description": "Return value from the `useFormControlUnstyledContext` hook."
+      },
+      "required": true
+    },
+    "getInputProps": {
+      "type": {
+        "name": "<TOther extends Record<string, any> = {}>(externalProps?: TOther) => UseInputInputSlotProps<TOther>",
+        "description": "Resolver for the input slot's props."
```

```suggestion
        "description": "Resolver for the `input` slot's props."
```

To be consistent with the other descriptions, where "input" is shown as a code element.